Sources/Fuzzilli/Protobuf/program.pb.swift

```diff
@@ -132,777 +132,799 @@ public struct Fuzzilli_Protobuf_Instruction {
   /// The operation is either encoded as an index, referring to the nth operation
   /// (so that shared operations are also only present once in the protobuf), or
   /// as one of the many concrete Operation messages.
-  public var inouts: [UInt32] = []
+  public var inouts: [UInt32] {
```

Comment:
Are all of these changes due to a newer version of the Swift protobuf compiler? I'd slightly prefer to do that in a separate PR if possible, but it's fine if that's too much hassle (e.g. getting an older version to work)

Reply:
Yeah, I've updated my protobuf compiler so there's a whole load of this changes.

Reply:
Hmm are you sure that's the latest release? For me I have
```
> protoc-gen-swift --version
protoc-gen-swift 1.18.0
```
which seems to be the latest version according to https://github.com/apple/swift-protobuf

Reply:
AH! Turns out I picked the last listed tag to compile from which was (1.9.0) and that appears after 1.18.0. I'll recompile and rebase my PRs :(